src/otx/algo/classification/backbones/otx_efficientnet_v2.py

```diff
@@ -1,6 +1,6 @@
 # Copyright (C) 2023 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
-#
```

Comment:
@harimkang, It seems that we don't have code formatter in our pre-commit rules. I have been able to see this kind of code clutters from other guys' PR (including me). If this is true, please introduce it.

Reply:
Yes, I get it, pre-commit doesn't seem to catch this stuff (file docstring).
I'll see if there's a tool that fixes it all at once, and if so, I'll do it on another PR.